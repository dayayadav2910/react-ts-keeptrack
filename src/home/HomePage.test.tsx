import { render, screen } from '@testing-library/react';
import HomePage from './HomePage';
import renderer from 'react-test-renderer';
// import { shallow, ShallowWrapper } from 'enzyme';

describe('<HomePage />', ()=>{
    test('Snapshot',()=>{
        const tree =  renderer.create(<HomePage></HomePage>).toJSON();
        expect(tree).toMatchSnapshot();
    });

    test('Renders Home Page',()=>{
    render(<HomePage></HomePage>)
    expect(screen.getByRole('heading')).toHaveTextContent('HomePage');
    });
})

