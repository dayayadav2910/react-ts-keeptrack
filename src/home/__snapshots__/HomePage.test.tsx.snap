// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<HomePage /> Snapshot 1`] = `
<div>
  <h2>
    HomePage
  </h2>
</div>
`;
